dash/dash.py

```diff
@@ -265,6 +298,8 @@ def __init__(
         name=None,
         server=True,
         assets_folder="assets",
+        pages_folder="pages",
+        pages=False,
```

Comment:
Good suggestion.  I changed it to `use_pages` for now, but I think @alexcjohnson was looking for a way to do this without using another attribute.  (but I'm not sure how to do that, so I'll leave this open)

Reply:
It could just be check for `os.path.exists(pages_folder)`, but I think for now it's better to have this flag for backward compatibility as users may have a `pages` directory with another multi pages pattern. It could be removed later on in a major version.

Reply:
Yeah I think the way you have it now is about as minimal as we can get: either `use_pages=True` or `pages_folder=<something_nondefault>` to enable the feature. So let's keep it this way.